### PR TITLE
release(chart): add helm chart changes for v3.4.0

### PR DESCRIPTION
**Pull request by niladrih**
Signed-off-by: Niladri Halder <niladri.halder26@gmail.com>

This adds helm chart changes for OpenEBS v3.4.0 release.

### CODE REVIEW

File: charts/openebs/Chart.yaml [hunk 1]
```diff
@@ -28,15 +28,15 @@ dependencies:
     repository: "https://openebs.github.io/node-disk-manager"
     condition: openebs-ndm.enabled
   - name: localpv-provisioner
-    version: "3.3.0"
+    version: "3.4.0"
```
Comment (Ab-hishek) [inline charts/openebs/Chart.yaml:31]: What about ndm, lvm-localpv & zfs-localpv?

Comment (niladrih) [reply charts/openebs/Chart.yaml:31]: They are not being released. Just cStor Jiva and Dynamic LocalPV.

Comment (Ab-hishek) [reply charts/openebs/Chart.yaml:31]: Why is this so. We usually release everything together?

**Review by Ab-hishek — APPROVED** [review]
LGTM

**Comment by ianroberts** [comment]
Given this PR was approved on 3rd October, any likely timeline on when the 3.4.0 charts will actually be released?  OpenEBS cStor 3.3.x and earlier are unusable with k8s 1.25 due to the use of the v1beta1 PDB API.

**Pull request merged**